### ISSUE TITLE
Allow open multiple pdf files

### ISSUE THREAD

**Issue by Augusto7743** [troyeguo/koodo-reader#342]
### I've read Koodo Reader's roadmap, there is no similar feature.

- [X] Yes

### This issue only requests one feature.

- [X] Yes

### Brief Introduction

Tested in Linux.

### Details

When loaded an pdf file if the user need open another pdf file using file managers clicking on file is done disk activity and in Koodo Reader window freeze for few seconds, but not open another pdf file and continue displaying the same previous pdf loaded.
In other words allow open multiple pdf at same time in each koodo reader window and if possible grouped in koodo reader main window.
Some times need to open several pdf. The user need load a file close Koodo Reader and repeat the same action to each next pdf.
Thanks for reply.

**Comment by troyeguo** [comment]
<img width="739" alt="捕获" src="https://user-images.githubusercontent.com/13820674/151356235-81ccc9a3-2a15-43b1-b5bf-da0a6a4ebe94.PNG">

**Comment by Augusto7743** [comment]
Thanks for reply.
Here strangely Koodo Reader only open pdf if opening books in the main window is being turned on.
If opening books in the main window is being turned off happen disk activity and Koodo Reader is started and after is opened another Koodo Reader window to pdf, but the window is blank without anything detail about the pdf.

Look how being an issue.
Using Koodo.Reader-1.3.9 in Linux Lubuntu (Ubuntu using LXQT) 20.04 64 bits.
